packages/cli/src/utils/directory.ts

```diff
@@ -18,50 +18,75 @@ export const withBasePath = (basepath: string) => {
     return path.resolve(process.cwd(), basepath)
   }
 
-  /* 
+  /*
    * This will loop from the basepath until the process.cwd() looking for node_modules/@faststore/core
-   * 
+   *
    * If it reaches process.cwd() (or /, as a safeguard), without finding it, it will throw an exception
    */
   const getCorePackagePath = () => {
-    const coreFromNodeModules = path.join('node_modules', '@faststore', 'core')
+    const packageFromNodeModules = path.join(
```

Comment:
Why was this variable renamed? If I understood correctly, most changes on this file are cosmetic (indentation) and you're adding the `getPackagePath` entry to the returned object, right? If that's so can you undo the rename of this `const`?

Reply:
Good, it must have been a mistake. Renamed.